### ISSUE TITLE
Accessibility Issue v10.6.1 - Umbraco Login Page - Logo does not have alt attribute

### ISSUE THREAD

**Issue by Truealex4** [umbraco/Umbraco-CMS#15069]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

v10.6.1

### Bug summary

When I access the login screen and run the accessibility lighthouse tool it identifes an issue with the logo where there is a missing alt attribute

### Specifics

/umbraco#/login/false?returnPath=%252Fumbraco%2523%252Fcontent
Affects all browsers

### Steps to reproduce

1. Access umbraco site on v10.6.1
2. Without logging in run the lighthouse accessibility report for Desktop

### Expected result / actual result

EXPECTED RESULT - No issues found
ACTUAL RESULT -The following issue was found:
![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/7f3a0e11-9326-41e3-9ff7-dd6ebbda3b1c)

**Comment by andr317c** [comment]
Thanks for posting this issue, I was able to reproduce it on versions 10.7.0 and 12.2.0. This is something we would like some help with, so I'll go ahead and set this issue up for grabs 😄

**Comment by mehrdadab** [comment]
Hi @andr317c
I would like to work on this issue.

**Comment by nul800sebastiaan** [comment]
Closed for the reason mentioned here: https://github.com/umbraco/Umbraco-CMS/pull/15182#issuecomment-1831863196 (in short, it's a decorative image, screen readers should skip it).
